specs/_features/maxeb_increase/capella.py

```diff
@@ -1316,7 +1359,7 @@ def get_unslashed_attesting_indices(state: BeaconState,
     output = set()  # type: Set[ValidatorIndex]
     for a in attestations:
         output = output.union(get_attesting_indices(state, a.data, a.aggregation_bits))
-    return set(filter(lambda index: not state.validators[index].slashed, output))
+    return set(filter(lambda index: not is_slashed_attester(state.validators[index]), output))
```

Comment:
I think we should have `is_slashed_validator(validator): return is_slashed_attester(validator) or is_slashed_proposer(validator)`, and use that here. Don't see why attestations from someone slashed as a proposer should count here